### DISCUSSION TITLE
Rendering Tuplets/ specifically triplets

### DISCUSSION THREAD

**Discussion by ElectricCy** [vexflow/vexflow#231] [Q&A]
Hello, 

I am trying to create an application that hosts an AI that uses VexFlow to create and display scales/ full on melodies, with proper rhythms. So far it is working great but I'm running into an issue trying to get triplets to work.

I'm an amateur so I'm really hoping someone can help me out here so I can continue making this cool application.

https://play.rosebud.ai/games/14f5023a-c271-48b0-8d01-8b30d47cf1bb

ask the AI to create a C major scale or any other scales, it works just fine. or asking it to do 8th notes, but triplets are giving me a hard time. I am able to play the notes with the midi player, but for some reason the staff is not being loaded with the triplets.

**Comment by rvilarl** [comment]
Hello,

Have you look into the example below? Can you share your example? 

https://github.com/vexflow/vexflow/blob/be0e0dbaeaa72a054f6c8298b374d7593d2b06d8/tests/tuplet_tests.ts#L43-L75
